src/components/Popup.component.tsx

```diff
@@ -98,7 +98,7 @@ export default function Popup({
               >
                 <Dialog.Panel className="w-full max-w-sm transform overflow-hidden rounded-2xl inv-background text-left align-middle shadow-xl transition-all">
                   <div className="flex flex-col items-center gap-2">
-                    <Slot item={recipes[solution].output} clickable={false} />
+                    <CraftingTable solved={true} />
```

Comment:
I think we should hide the border of the crafting table within the popup.

Can do this by adding an optional attribute `hideBorder` that defaults to `false` in CraftingTable.component that hides the border.  Then pass `hideBorder={true}` here to hide the border in the popup.